pandas/tests/resample/test_period_index.py

```diff
@@ -877,3 +891,53 @@ def test_sum_min_count(self):
             [3.0, np.nan], index=PeriodIndex(["2018Q1", "2018Q2"], freq="Q-DEC")
         )
         tm.assert_series_equal(result, expected)
+
+
+def test_period_index_frequency_ME_error_message():
+    msg = "Invalid frequency: 2ME"
+
+    with pytest.raises(ValueError, match=msg):
+        PeriodIndex(["2020-01-01", "2020-01-02"], freq="2ME")
```

Comment:
likewise, there's no resampling here, can we move it to a periodindex-specific place?